common/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     api 'com.squareup.moshi:moshi:1.3.1'
     api 'com.squareup.retrofit2:retrofit:2.3.0'
     api 'com.squareup.retrofit2:converter-moshi:2.3.0'
+    api 'com.squareup.retrofit2:converter-scalars:2.3.0'
```

Comment:
We need this to handle the response from the revoke command, which returns text and not json.